### PR TITLE
Readme examples new features

### PR DESCRIPTION
**Pull request by OwenElliottDev**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Documentation

* **What is the current behavior?** (You can also link to an open issue here)
There **aren't** examples for multimodal combination fields and queries with weights in the README

* **What is the new behavior (if this is a feature change)?**
There **are** examples for multimodal combination fields and queries with weights in the README that match the documentation

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)
README.md renders correctly

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,84 +215,221 @@ result = mq.index("my-first-index").search('adventure', searchable_attributes=['
...
-### Delete index
-
-Delete an index.
+Images can then be added within documents as follows. You can use urls from the internet (for example S3) or from the disk of the machine:
 
 ```python
 
-results = mq.index("my-first-index").delete()
+response = mq.index("my-multimodal-index").add_documents([{
+    "My Image": "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b3/Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg/640px-Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg",
```
Comment (pandu-k) [inline README.md]: Wikipedia images, like this, aren't very stable in my experience. We have these Hippo images that are quite useful:
- realistic: `https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_realistic.png`
- artefact: `https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_statue.png`

File: README.md [hunk 2]
```diff
@@ -215,84 +215,221 @@ result = mq.index("my-first-index").search('adventure', searchable_attributes=['
...
+Searching using an image can be achieved by providing the image link.
 
 ```python
 
-response = mq.index("my-multimodal-index").add_documents([{
-    "My Image": "https://upload.wikimedia.org/wikipedia/commons/thumb/f/f2/Portrait_Hippopotamus_in_the_water.jpg/440px-Portrait_Hippopotamus_in_the_water.jpg",
-    "Description": "The hippopotamus, also called the common hippopotamus or river hippopotamus, is a large semiaquatic mammal native to sub-Saharan Africa",
-    "_id": "hippo-facts"
-}])
+results = mq.index("my-multimodal-index").search('https://upload.wikimedia.org/wikipedia/commons/thumb/9/96/Standing_Hippopotamus_MET_DP248993.jpg/1920px-Standing_Hippopotamus_MET_DP248993.jpg')
```
Comment (pandu-k) [inline README.md]: same comment as above regarding wikipedia image stability

File: README.md (hunk 1 above)
Comment (OwenElliottDev) [reply README.md]: That section is a copy-paste from the current README, I will test and update it with one of the more stable links though as I also agree that they would be better.

File: README.md (hunk 2 above)
Comment (OwenElliottDev) [reply README.md]: Same as my reply to your other comment

File: README.md (hunk 1 above)
Comment (OwenElliottDev) [reply README.md]: @pandu-k I have added a commit to use the image links you provided

**Comment by jn2clark** [comment]
do the tests need to be updated @pandu-k ? that can probably happen as a second PR? actually, it should probably just happen with this

**Comment by pandu-k** [comment]
Good point @jn2clark . @OwenPendrighElliott , can you create a Py-Marqo test that ensures that the new README example works as intended? See [here](https://github.com/marqo-ai/py-marqo/blob/mainline/tests/v0_tests/test_demos.py#L55) for example.

**Comment by OwenElliottDev** [comment]
@pandu-k see marqo-ai/py-marqo#78

**Review by pandu-k — APPROVED** [review]

**Pull request merged**